shared/common-adapters/avatar.desktop.js

```diff
@@ -58,7 +58,7 @@ class Avatar extends PureComponent<void, Props, State> {
             ...avatarStyle,
             ...borderStyle,
             display: (!showNoAvatar && !showLoadingColor) ? 'block' : 'none',
-            backgroundColor: globalColors.white,
+            backgroundColor: this.props.backgroundColor || globalColors.white,
```

Comment:
if this is white always you can get bad aliasing on the edges on dark backgrounds due to the alpha blend on the border radius